### PR TITLE
MAINT: try fix cis

### PR DESCRIPTION
**Pull request by alexrockhill**
Fix the CI failures

### CODE REVIEW

**Comment by alexrockhill** [comment]
Ok @larsoner , sorry I'm a bit stuck on this one too, do you know how to increase the read timeout on MNE dataset fetching?

image

**Comment by larsoner** [comment]
No I haven't hit an issue... you could look to see if there is some global var but from the looks of it we use `pooch` so you'd have to see what their default is etc.

Also in MNE-Python we download all datasets *before* running any examples. It can help clean up the captured output and more accurately reflect the example timing.

**Comment by alexrockhill** [comment]
It looks like pooch hardcodes it, I will trying downloading before

**Comment by alexrockhill** [comment]
Hey @larsoner sorry to bug you again but I was trying to read this thread an suppress the warning but I couldn't figure out how to get the scraper to pickle like it's done in `mne-python`, mind pointing me in the right direction?

**Comment by larsoner** [comment]
The easy workaround on CIs is https://github.com/mne-tools/mne-python/pull/12552 (pin sphinx) or https://github.com/mne-tools/mne-python/pull/12548/files#diff-e170e9a7d787c21095c6c11bb25f0f1ff0294a42a46d45ba6fb5ed794e457624R809 (ignore the warning). The correct long-term fix is https://github.com/mne-tools/mne-python/pull/12574 but requires latest SG `master` (but should be released next week)

**Comment by alexrockhill** [comment]
I tried to implement the long term fix, maybe I just need the latest sg for the scapers to get recognized, I'll check

**Comment by alexrockhill** [comment]
Ok by you @larsoner? Finally everything passes

File: .circleci/config.yml [hunk 1]
```diff
@@ -42,7 +42,8 @@ jobs:
         - run:
             name: Get Python running
             command: |
-              pip install --upgrade PyQt6!=6.6.1 "PyQt6-Qt6!=6.6.1" sphinx-gallery pydata-sphinx-theme numpydoc scikit-learn nilearn mne-bids autoreject git+https://github.com/pyvista/pyvista@main memory_profiler sphinxcontrib.bibtex sphinxcontrib.youtube darkdetect qdarkstyle
+              # TO DO: sphinx-gallery main -> stable on 0.17 release
+              pip install --upgrade PyQt6!=6.6.0 "PyQt6-Qt6!=6.6.0,!=6.7.0" git+https://github.com/sphinx-gallery/sphinx-gallery.git pydata-sphinx-theme numpydoc scikit-learn nilearn mne-bids autoreject pyvista memory_profiler sphinxcontrib.bibtex sphinxcontrib.youtube darkdetect qdarkstyle
```
[the remark this answers is not in the document]
Comment (alexrockhill) [reply .circleci/config.yml:46]: Ah my bad! I'll take it out in the #27

**Pull request merged**